src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -2230,16 +2232,16 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
                     target->ToUnit()->RemoveAllGameObjects();
             break;
         }
-        case SMART_ACTION_STOP_MOTION:
+        case SMART_ACTION_REMOVE_MOVEMENT:
```

Comment:
Is the change to this action backwards compatible? If not then you need to update db data to keep behavior the same

Reply:
there is not, currently, any script using the action

> backwards compatible?
I guess so... the old one make 0 sense and hardly ever worked